eng/Versions.props

```diff
@@ -53,6 +54,7 @@
     <SystemValueTuplePackageVersion>4.5.0</SystemValueTuplePackageVersion>
   </PropertyGroup>
   <ItemGroup>
+    <DotNetRuntime Include="$(MicrosoftNETCoreApp21PackageVersion)" />
```

Comment:
Does `--roll-forward-on-no-candidate-fx 2` not work?

Reply:
It does leftover I forgot about.